apps/condo/domains/common/utils/mutations.utils.js

```diff
@@ -73,15 +73,15 @@ function runMutation ({ action, mutation, variables, onCompleted, onError, onFin
     return action()
         .then(
             (data) => {
-                if (OnCompletedMsg === null) {
+                if (getCompletedNotification === null) {
```

Comment:
We still have some places in code where we are using OnCompletedMsg. Maybe, you can add one more check if OnCompletedMsg is a function ? It will be more safe then remove old variable and add another